pep2html.py

```diff
@@ -520,32 +522,32 @@ def main(argv=None):
             browse = 1
 
     if args:
-        peptxt = []
+        peptxtrst = []
```

Comment:
I'd rename it to `peps`, `pep_list` or something else (`peptxtrst` won't make sense once we convert all peps to reST).